content/800-data-platform/200-projects/100-add-project/120-import-existing-project-existing-db.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Import an existing project with an existing database'
+metaTitle: 'Import an existing JavaScript/TypeScript project with an existing database in the Prisma Data Platform'
+metaDescription: 'You can import your production application in the Prisma Data Platform and configure the Data Proxy to enable connection pooling and avoid exhausting the database connections in serverless environments.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Import your production application and configure the [Data Proxy](/data-platform/data-proxy) to enable connection pooling and avoid exhausting the database connections in serverless environments.
+
+</TopBlock>
+
+## Prerequisites and considerations
+
+* Log in to GitHub with your account ([or sign up for one](https://github.com/signup)).
+* Prepare a GitHub repository with your JavaScript/TypeScript application that uses Prisma ORM.
+* Obtain a connection string to a [Prisma ORM-supported database](/reference/database-reference/supported-databases).
+* [Sign up for the Prisma Data Platform](/data-platform/account#sign-up-for-the-prisma-data-platform).
+* [Grant the Prisma GitHub app read and write permissions to your GitHub repositories](/data-platform/projects/add-project#grant-read-and-write-permissions-to-the-prisma-github-app).
+
+## Steps
+
+1. Go to the [_Projects_](https://cloud.prisma.io/projects) page and click **New Project**.
```

Comment:
I noticed that you consistently format page titles and section headings in italics when referring to them. IMO, these also count as [UI elements](https://www.prisma.io/docs/about/prisma-docs/style-guide/spelling-punctuation-formatting#ui-elements) and therefore should be formatted in bold font as well.